feat(dataarts): add new datasource for query app authorized apis (#9402)
huaweicloud/terraform-provider-huaweicloud: docs/data-sources/dataarts_dataservice_app_authorized_apis.md

---
subcategory: "DataArts Studio"
layout: "huaweicloud"
page_title: "HuaweiCLoud: huaweicloud_dataarts_dataservice_app_authorized_apis"
description: |-
  Use this data source to get the list of authorized APIs under specified APP within HuaweiCloud.
---

# huaweicloud_dataarts_dataservice_app_authorized_apis

Use this data source to get the list of authorized APIs under specified APP within HuaweiCloud.

## Example Usage

```hcl
variable "workspace_id" {}
variable "app_id" {}

data "huaweicloud_dataarts_dataservice_app_authorized_apis" "test" {
  workspace_id = var.workspace_id
  app_id       = var.app_id
}
```

## Argument Reference

The following arguments are supported:

> * `region` - (Optional, String) Specifies the region where the APP is located.  

Reviewer: ```suggestion
* `region` - (Optional, String) Specifies the region where the APIs are located.  
```

Reviewer 2: authorized APIs are located

Author: done
